python/import_util.py

```diff
@@ -122,13 +128,244 @@ def json(ctx: mgp.ProcCtx, path: str) -> mgp.Record():
             )
         else:
-            raise KeyError("The provided file does not match the correct JSON format.")
+            raise KeyError(
+                "The provided file does not match the correct JSON format."
+            )
 
     return mgp.Record()
+
+
+def find_node(
+    ctx: mgp.ProcCtx, label: str, prop_key: str, prop_value: str
+) -> int:  # TODO prop value cast??
+    for vertex in ctx.graph.vertices:
+        if (
+            label in [label.name for label in vertex.labels]
+            and prop_key in vertex.properties.keys()
+            and str(convert_to_isoformat_graphML(vertex.properties.get(prop_key)))
+            == prop_value
+        ):
+            return vertex.id
+    return None
+
+
+def cast_element(
+    text: str, type: str
+) -> Union[List[Any], str, int, bool, float]:
+    if text is None:
+        return ""
+    if type == "string":
+        return str(text)
+    if type == "int" or type == "long":
+        return int(text)
+    if type == "boolean":
+        return bool(text)
+    if type == "float" or type == "double":
+        return float(text)
+    if type is None:
+        return text
+
+
+def cast(
+    text: str, type: str, is_list: str
+) -> Union[List[Any], str, int, bool, float]:
+    if is_list is not None:
+        casted_list = list()
+        for element in ast.literal_eval(text):
+            casted_list.append(cast_element(element, is_list))
+        return casted_list
+    return cast_element(text, type)
+
+
+def set_default_keys(key_dict: Dict[str, Any], properties: Dict[str, Any]):
+    for key, value in key_dict.items():
+        if value[3] is not None:
+            properties.update({value[0]: cast(value[3], value[1], value[2])})
+
+
+def set_default_config(config: mgp.Map):
+    if not config.get("readLabels"):
+        config.update({"readLabels": False})
+    if not config.get("defaultRelationshipType"):
+        config.update({"defaultRelationshipType": "RELATED"})
+    if not config.get("storeNodeIds"):
+        config.update({"storeNodeIds": False})
+    if not config.get("source"):
+        config.update({"source": {}})
+    if not config.get("target"):
+        config.update({"target": {}})
+    if (
+        not isinstance(config.get("readLabels"), bool)
+        or not isinstance(config.get("defaultRelationshipType"), str)
+        or not isinstance(config.get("storeNodeIds"), bool)
+        or not isinstance(config.get("source"), dict)
+        or not isinstance(config.get("target"), dict)
+        or (
+            config.get("source") and "label" not in config.get("source").keys()
+        )
+        or (
+            config.get("target") and "label" not in config.get("target").keys()
+        )
+    ):
+        raise TypeError(
+            "Config parameter must be a map with specific \
+             keys and values described in documentation."
+        )
+
+
+@mgp.write_proc
+def graphml(
+    ctx: mgp.ProcCtx,
+    path: str = "",
+    config: mgp.Map = {},
```

Comment:
```suggestion
    config: mgp.Map = dict(),
```
Using an empty dictionary as a default parameter may cause the module to act unpredictably because that variable will persist until the next call ([source](https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python)).